src/options.ts

```diff
@@ -394,7 +394,7 @@ export interface ContainsOptions {
 }
 
 export interface IsAlphaOptions {
-  ignore?: string[] | RegExp;
+  ignore?: string | RegExp;
 }
```

Comment:
This is potentially a breaking change!

Reply:
Indeed. A way to make it not be breaking would be to test the type of `ignore` in the `isAlpha` implementation, WDYT?

Reply:
Yes, I was thinking about that.